docs/modules/operation/pages/quick-start/introduction.adoc

```diff
@@ -1,7 +1,35 @@
 
 = Quick Start
 
-The Quick Start guide assumes you have already completed the deployment steps, have installed a xref:deployment:core/introduction.adoc[{page-component-title} instance], and are able to connect to the web UI at `\http://<ip-or-fqdn-of-your-server>:8980/opennms`.
+This Quick Start guide assumes that you have already completed the xref:deployment:core/introduction.adoc[deployment steps], have xref:deployment:core/getting-started.adoc[installed a {page-component-title} instance], and that you are able to connect to the web UI at `\http://<ip-or-fqdn-of-your-server>:8980/opennms`.
```

Comment:
Let's add a sentence here that explicitly states that it includes the steps to set up basic monitoring with OpenNMS with minimal configuration. More in-depth information on the topics including more complex configuration is available in the Deep Dive section.